demo-app/src/main/kotlin/io/getstream/video/android/ui/call/CallScreen.kt

```diff
@@ -928,6 +928,7 @@ private fun BadNetworkLabel(
 
 @Composable
 fun isTablet(): Boolean {
+    return true
```

Comment:
This was done only for testing

Reply:
Should we remove this check al together in demo app @rahul-lohra @aleksandar-apostolov ?